addons/mail/static/tests/m2x_avatar_user_tests.js

```diff
@@ -58,189 +58,57 @@ QUnit.module('mail', {}, function () {
-            // 'call service openDMChatWindow 1',
-        ]);
+        assert.strictEqual(
+            document.querySelector('.o_ChatWindowHeader_name').textContent,
+            'Partner 1',
+            'Chat window should be related to partner 1'
+        );
 
         list.destroy();
     });
 
-    QUnit.test('many2one_avatar_user widget in kanban view', async function (assert) {
-        assert.expect(6);
-
-        const { widget: kanban } = await start({
-            hasView: true,
-            View: KanbanView,
-            model: 'foo',
-            data: this.data,
-            arch: `
-                <kanban>
-                    <templates>
-                        <t t-name="kanban-box">
-                            <div>
-                                <field name="user_id" widget="many2one_avatar_user"/>
-                            </div>
-                        </t>
-                    </templates>
-                </kanban>`,
-        });
-
-        assert.strictEqual(kanban.$('.o_kanban_record').text().trim(), '');
-        assert.containsN(kanban, '.o_m2o_avatar', 4);
-        assert.strictEqual(kanban.$('.o_m2o_avatar:nth(0) > img').data('src'), '/web/image/res.users/11/avatar_128');
-        assert.strictEqual(kanban.$('.o_m2o_avatar:nth(1) > img').data('src'), '/web/image/res.users/7/avatar_128');
-        assert.strictEqual(kanban.$('.o_m2o_avatar:nth(2) > img').data('src'), '/web/image/res.users/11/avatar_128');
-        assert.strictEqual(kanban.$('.o_m2o_avatar:nth(3) > img').data('src'), '/web/image/res.users/23/avatar_128');
-
-        kanban.destroy();
-    });
-
     QUnit.test('many2many_avatar_user widget in form view', async function (assert) {
-        assert.expect(7);
+        assert.expect(2);
 
         const { widget: form } = await start({
+            hasChatWindow: true,
             hasView: true,
             View: FormView,
             model: 'foo',
             data: this.data,
             arch: '<form><field name="user_ids" widget="many2many_avatar_user"/></form>',
-            mockRPC(route, args) {
-                if (args.method === 'read') {
-                    assert.step(`read ${args.model} ${args.args[0]}`);
-                }
-                return this._super(...arguments);
-            },
             res_id: 1,
         });
 
-        assert.containsN(form, '.o_field_many2manytags.avatar.o_field_widget .badge', 2,
-            "should have 2 records");
-        assert.strictEqual(form.$('.o_field_many2manytags.avatar.o_field_widget .badge:first img').data('src'), '/web/image/res.users/11/avatar_128',
-            "should have correct avatar image");
-
         await dom.click(form.$('.o_field_many2manytags.avatar .badge:first .o_m2m_avatar'));
         await dom.click(form.$('.o_field_many2manytags.avatar .badge:nth(1) .o_m2m_avatar'));
+        assert.containsN(document.body, '.o_ChatWindow', 2, '2 Chat windows should be opened');
```

Comment:
I don't think click on 2 different avatar and have 2 chat windows is relevant for this test. Could just test with a single click => single chat window so that test is much simpler.